day-2/rabbitmq/service-monitor.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
According to the helm chart, the monitoring is already included. Only needs to be enabled

https://github.com/bitnami/charts/blob/main/bitnami/rabbitmq-cluster-operator/values.yaml#L457

Reply:
According to the `values.yaml`, monitoring is already enabled. However I cannot any metric related to RabbitMQ on prometheus